.github/workflows/ci-lite-test-full.yml

```diff
@@ -106,19 +111,18 @@ jobs:
 
     - name: Statistics
       if: success()
-      working-directory: tests/tests_lite
+      working-directory: ./tests
       run: |
         coverage report
         coverage xml
 
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v3
-      if: always()
```

Comment:
Is this removed on purpose, why?

Again, I suggest these coverage related changes are done in a separate PR for all workflows.

Reply:
yes, because if pytest/workflow fails, we shall not run this step because there is not report... and if you check it is not in other workflows